test/topology_custom/test_tablets.py

```diff
@@ -35,8 +35,10 @@ async def test_tablet_replication_factor_enough_nodes(manager: ManagerClient):
     with pytest.raises(ConfigurationException, match=f"Datacenter {this_dc} doesn't have enough nodes"):
         await cql.run_async("CREATE TABLE test.test (pk int PRIMARY KEY, c int);")
 
-    await cql.run_async(f"ALTER KEYSPACE test WITH replication = {{'class': 'NetworkTopologyStrategy', '{this_dc}': 2}}")
-    await cql.run_async("CREATE TABLE test.test (pk int PRIMARY KEY, c int);")
+    with pytest.raises(InvalidRequest, match="Changing replication options via ALTER KEYSPACE statement is not yet supported for keyspaces with tablets."):
```

Comment:
As usual, you're breaking the test to enshrine some temporarily-broken behavior, without even a comment reminding us to fix this. I would have just used xfail here.

Reply:
Reverted the changes and marked with xfail instead.